kubectl-pg/cmd/version.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"k8s.io/client-go/kubernetes"
+	"log"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+const kubectlPgVersion = "0.1-beta"
```

Comment:
That should not be hard-coded by instead added at build time. Check out a couple of approaches: [one](https://blog.polyverse.io/how-to-embed-versioning-information-in-go-applications-f76e2579b572), [two](https://www.reddit.com/r/golang/comments/4cpi2y/question_where_to_keep_the_version_number_of_a_go/)